.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,12 +32,16 @@ Fixes #ISSUE
 - [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] Any dependent changes have been merged
-- [ ] I have written Kotlin Docs whenever is applicable
 
 
 **Code/Quality Assurance Only**
 - [ ] My changes generate no new warnings
 - [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been published in downstream modules
\ No newline at end of file
+- [ ] Any dependent changes have been published in downstream modules
+
+**Desktop (please complete the following information):
```

Comment:
No need for desktop and browser details to be included in PR templates, please remove this